frontend/src/components/samples/SamplesDetailContent.js

```diff
@@ -164,6 +177,25 @@ const SamplesDetailContent = ({samplesByID, sampleKindsByID, containersByID, ind
 
       <TrackingFieldsContent entity={sample}/>
 
+      <Title level={2} style={{ marginTop: '1em' }}>Processes</Title>
+      <Row>
+        <Col sm={24} md={24}>
+          <div>
+            <Card>
+              {
+                isProcessesEmpty ?
+                  <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} /> :
+                  <div>
+                      {sampleProcesses &&
```

Comment:
Why `sampleProcesses && ...`? First, that thing is an array so it's always truthy. Second, aren't you already checking that it's not empty?